internal/provider/provider.go

```diff
@@ -132,6 +132,7 @@ func Provider() *schema.Provider {
 			"packetfabric_marketplace_service_requests":  dataSourceVcRequests(),
 			"packetfabric_virtual_circuits":              datasourceBackboneServices(),
 			"packetfabric_point_to_point":                datasourcePointToPoint(),
+			"packetfabric_locations_regions":             datasourceLocationsRegions(),
```

Comment:
I just noticed the name of the function isn't the same as some other: shouldn't it be `dataSourceLocationsRegions` instead of `datasourceLocationsRegions`.

Same for `datasourceBackboneServices` and `datasourcePointToPoint`

Reply:
you're right, I added this to this other issue we have in the backlog to cleanup the code https://github.com/PacketFabric/terraform-provider-packetfabric/issues/191